opds.py

```diff
@@ -1179,9 +1179,27 @@ def license_tags(cls, license_pool, loan, hold):
             return tags
 
 
-        holds_kw = dict(total=str(license_pool.patrons_in_hold_queue or 0))
-        if hold and hold.position:
-            holds_kw['position'] = str(hold.position)
+        holds_kw = dict()
+        total = license_pool.patrons_in_hold_queue or 0
+        position = hold.position or 0
```

Comment:
I'm not sure when we'd be missing the hold position, but it might be safer to assume you're last rather than first if we don't know.